<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>三维信息重构软件</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="CVWidget" name="cvViewer">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>411</width>
      <height>491</height>
     </rect>
    </property>
    <property name="text">
     <string>camera ERROR</string>
    </property>
   </widget>
   <widget class="pgWidget" name="graphwidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>481</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>730</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>开始扫描</string>
    </property>
   </widget>
   <widget class="QPushButton" name="surfaceButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>730</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string> 表面重建</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calibrateButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>730</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>相机标定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>540</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>540</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="serialLabel">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>540</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>无串口</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>600</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="scanprogressBar">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>600</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>740</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>740</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>计算所用核心数</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>720</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>计算机核心总数：</string>
    </property>
   </widget>
   <widget class="QLabel" name="CPULabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>720</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>nan</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>580</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>扫描进度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>580</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>提取进度</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>730</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>导出模型</string>
    </property>
   </widget>
   <widget class="QVTKRenderWindowInteractor" name="vtkWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>481</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>640</y>
      <width>391</width>
      <height>121</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>964</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pgWidget</class>
   <extends>QWidget</extends>
   <header>pgWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CVWidget</class>
   <extends>QLabel</extends>
   <header>cvWidget</header>
  </customwidget>
  <customwidget>
   <class>QVTKRenderWindowInteractor</class>
   <extends>QWidget</extends>
   <header>vtk.qt.QVTKRenderWindowInteractor</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
